modules/ROOT/pages/Getting_started_with_Bonita_Cloud.adoc

```diff
@@ -5,7 +5,7 @@
 With Bonita Cloud starting a new project has never been easier:
 
 . After receiving all the required technical credentials, xref:{bonitaDocVersion}@bonita:runtime:first-steps-after-setup.adoc[create a Bonita Portal administrator for your runtimes]
-. Download the Bonita Studio in your customer Portal (https://customer.bonitasoft.com/download/request[Request a download]).
+. Download the Bonita Studio in your customer Portal (https://csc.bonitacloud.bonitasoft.com/apps/CustomerServices[Request a download]).
```

Comment:
**question:** If someone know the exact URL to reach the request page (if it exists), please do a suggestion or commit the change directly 😸